general/templatetags/filter_img.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from bs4 import BeautifulSoup, Tag
+from os.path import basename, splitext
+
+register = template.Library()
+
+
+@register.filter(is_safe=True)
+def replace_img(string):
+    if not "<img" in string or not "http:" in string:
+        return string
+
+    soup = BeautifulSoup(string)
```

Comment:
bs4 should have a parser argument: https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser